ui/src/components/CardListItem.vue

```diff
@@ -1,21 +1,168 @@
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.employmentTypes) {
+               for (let i = 0; i < this.employmentTypes.length; i++) {
+                label.push(this.employmentTypes[i].key);
+                value.push(this.employmentTypes[i].doc_count);
+            }
+
+            const ctx = document.getElementById('employmentType-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Employment Types',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.skills) {
+               for (let i = 0; i < this.skills.length; i++) {
+                label.push(this.skills[i].key);
+                value.push(this.skills[i].doc_count);
+            }
+
+            const ctx = document.getElementById('skill-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Skills',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.isRemote) {
+               for (let i = 0; i < this.isRemote.length; i++) {
+                label.push(this.isRemote[i].key);
+                value.push(this.isRemote[i].doc_count);
+            }
+
+            const ctx = document.getElementById('remote-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Remote Eligible',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+    }
 }
 </script>
 
 <style scoped>
-
+    .cardList {
+        width: 80%;
+        margin: auto !important;
```

Comment:
Try not to use !important

Reply:
Removing !important
Find a way to remove style that was set by Semantic UI

Reply:
Actually this `.cardlist` class doesn't seem to be used so you can remove it

Reply:
It seems `fluid` class from semantic ui is forcing width of 100%. Is there any reason you want to use `fluid` class?

Reply:
Though if we use fluid then it takes up the width equally for each cards on given total width (container). Removed it and using flex-box.

Reply:
That can be done using flex box as well